docs/mta/README.md

```diff
@@ -31,17 +31,25 @@ done
 echo "https://"$(oc -n openshift-mta get route mta -o yaml | yq -r .spec.host)
 ```
 
+### Edit the `${WORKFLOW_NAME}-creds` Secret
+The token for sending notifications from the m2k workflow to RHDH notifications service needs to be provided to the workflow.
+
 To obtain the value for `NOTIFICATIONS_BEARER_TOKEN` use the value of the following command:
 ```bash
 oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET | base64decode }}'
 ```
 Or fetch the secret from the app-config.yaml of your RHDH instance if not installed by the Orchestrator operator.
 
-And to edit the configmap:
-```console
-oc -n <namespace> edit configmap mtaanalysis-props
+Now, edit the secret `${WORKFLOW_NAME}-creds` and set the value of `NOTIFICATIONS_BEARER_TOKEN`:
+```
+WORKFLOW_NAME=mtaanalysis
+oc -n sonataflow-infra patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{"data": { "NOTIFICATIONS_BEARER_TOKEN": "'$(echo -n "${NOTIFICATIONS_BEARER_TOKEN}" |base64 -w0)'"}}'
```

Comment:
I wonder if we should do tie the two commands to fetch the notification token and patch it into the secret.
First, it will reduce the need to decode the secret and encode it again (TBH, even now it isn't needed).

Alternately, instead of `"data": ` we can merge unencoded secret using `"stringData": `

Reply:
done